airmozilla/base/templates/email_base.html

```diff
@@ -1,47 +1,78 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html>
-
+ 
```

Comment:
Sublime text. I can clean it up and google around for settings to change to help with this. Thanks for letting me know!